stdlib/public/Platform/CMakeLists.txt

```diff
@@ -37,29 +37,25 @@ foreach(sdk ${SWIFT_SDKS})
     foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
       set(arch_subdir "${SWIFT_SDK_${sdk}_LIB_SUBDIR}/${arch}")
       set(module_dir "${SWIFTLIB_DIR}/${arch_subdir}")
-
-      # Determine the location of glibc based on the target.
-      if(${sdk} STREQUAL "ANDROID")
-        set(GLIBC_INCLUDE_PATH "${SWIFT_ANDROID_SDK_PATH}/usr/include")
-        set(GLIBC_ARCH_INCLUDE_PATH ${GLIBC_INCLUDE_PATH})
-      else()
-        set(GLIBC_INCLUDE_PATH "/usr/include")
-        set(GLIBC_ARCH_INCLUDE_PATH ${GLIBC_INCLUDE_PATH})
-        if((${sdk} STREQUAL "LINUX" OR ${sdk} STREQUAL "FREEBSD") AND CMAKE_LIBRARY_ARCHITECTURE)
-          # FIXME: Some distributions install headers in
-          #        "/usr/include/x86_64-linux-gnu/sys/...". Here we use the host
-          #        machine's path, regardless of the SDK target we're building for.
-          #        This will break if cross-compiling from a distro that uses the
-          #        architecture as part of the path to a distro that does not.
-          set(GLIBC_ARCH_INCLUDE_PATH "${GLIBC_INCLUDE_PATH}/${CMAKE_LIBRARY_ARCHITECTURE}")
-        endif()
+      
+      # Determine the location of glibc headers based on the target.
+      set(GLIBC_NATIVE_INCLUDE_PATH "/usr/include")
```

Comment:
Could you rename `NATIVE` => `SYSROOT_RELATIVE`?  I think that describes it better.